package.json

```diff
@@ -33,37 +38,28 @@
    "devDependencies": {
       "@silvermine/chai-strictly-equal": "1.1.0",
       "@silvermine/eslint-config": "3.1.0-beta.0",
-      "@silvermine/standardization": "1.2.1",
-      "@silvermine/typescript-config": "git+https://github.com/silvermine/typescript-config.git#23213e33077089e723629dead5342abe6f3b3c8c",
+      "@silvermine/standardization": "2.0.0",
+      "@silvermine/typescript-config": "git+https://github.com/silvermine/typescript-config#23213e33077089e723629dead5342abe6f3b3c8c",
```

Comment:
Just checking, do we need to remove the `.git`? I'm not seeing this change in the package lock, so I'm wondering if NPM is doing something fancy behind the scenes.

Reply:
This is what NPM resolves the package path to when you run  `npm i -DE git+https://github.com/silvermine/typescript-config/commit/23213e33077089e723629dead5342abe6f3b3c8c`